randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -592,8 +598,11 @@ Extra - room_id: [14]
   * Extra - door_idx: (28,)
   > Door to Forbidden Entrance
       Any of the following:
```

Comment:
you needing to do all those shenanigans makes me wonder whether we should have item requirements for can kill tough weak enemy here....